scripts/metadata_comparison/test/lib/test_operation_ids.py

```diff
@@ -60,22 +60,22 @@ def test_operation_id_number_valid(self):
                 self.assertEqual(get_operation_id_number(input), expectation)
 
     def test_find_v1alpha2_operation_ids_in_metadata(self):
-        metadata = json.loads(read_resource('forkjoin_metadata_papi_v1alpha2.json'))
+        metadata = json.loads((RESOURCES / 'forkjoin_metadata_papi_v1alpha2.json').read_text())
         self.assertEqual(find_operation_ids_in_metadata(metadata), self.v1alpha2_ids)
 
 
     def test_find_v2alpha1_operation_ids_in_metadata(self):
-        metadata = json.loads(read_resource('forkjoin_metadata_papi_v2alpha1.json'))
+        metadata = json.loads((RESOURCES / 'forkjoin_metadata_papi_v2alpha1.json').read_text())
         self.assertEqual(find_operation_ids_in_metadata(metadata), self.v2alpha1_ids)
 
 
     def test_find_v2beta_operation_ids_in_metadata(self):
-        metadata = json.loads(read_resource('forkjoin_metadata_papi_v2beta.json'))
+        metadata = json.loads((RESOURCES / 'forkjoin_metadata_papi_v2beta.json').read_text())
         self.assertEqual(find_operation_ids_in_metadata(metadata), self.v2beta_ids)
 
 
     def test_find_operation_ids_in_metadata_subworkflows(self):
-        metadata = json.loads(read_resource('subworkflow_hello_world_metadata.json'))
+        metadata = json.loads((RESOURCES / 'subworkflow_hello_world_metadata.json').read_text())
```

Comment:
@cjllanwarne I did not know you could simply do `<Path object> / <String>` and that that would also yield a Path object. But it makes sense as in Python you can override the division operator. This makes it even easier to use than I anticipated. Thanks!